CollapseLauncher/XAMLs/StartUp/StartupLanguageSelect.xaml

```diff
@@ -61,18 +64,20 @@
                             </Button.Content>
                             <Button.Flyout>
                                 <Flyout>
-                                    <StackPanel>
+                                    <StackPanel MaxWidth="380">
                                         <TextBlock>
-                                            <Run Text="What is a CDN?" FontWeight="Bold" FontSize="20"/>
-                                            <Run Text="short for" FontSize="12"/>
-                                            <Run Text="Content Delivery Network" FontWeight="Bold" FontSize="12"/>
+                                            <Run Text="{x:Bind helper:Locale.Lang._StartupPage.CDNHelpTitle_1}" FontWeight="Bold" FontSize="20"/>
+                                            <Run Text="{x:Bind helper:Locale.Lang._StartupPage.CDNHelpTitle_2}" FontSize="12"/><!--
+                                         --><Run Text="{x:Bind helper:Locale.Lang._StartupPage.CDNHelpTitle_3}" FontWeight="Bold" FontSize="12"/><!--
+                                         --><Run Text="{x:Bind helper:Locale.Lang._StartupPage.CDNHelpTitle_4}" FontSize="12"/>
```

Comment:
Note here: connect all Runs to avoid additional white space. For language that does now need word segmentation like Chinese.